openpype/tools/publisher/widgets/widgets.py

```diff
@@ -198,12 +205,24 @@ def __init__(self, parent=None):
         self.setLayoutDirection(QtCore.Qt.RightToLeft)
 
 
+class SaveBtn(PublishIconBtn):
+    """Save context and instances information."""
+    def __init__(self, parent=None):
+        icon_path = get_icon_path("save")
+        super(SaveBtn, self).__init__(icon_path, parent)
+        self.setToolTip(
+            "Save changes ({} + S)".format(get_control_str())
+        )
```

Comment:
Should we use Qt to define the human readable name for it?
```suggestion
        shortcut = QtGui.QKeySequence(QtGui.QKeySequence.Save).toString()
        self.setToolTip(
            "Save changes ({})".format(shortcut)
        )
```

We can do the same for the reset button:

```python
key_sequence =QtGui.QKeySequence(QtCore.Qt.ControlModifier | QtCore.Qt.Key_R)
shortcut = key_sequence.toString()
```

You could use this same key sequence to match the event instead of checking control modifier and save key separately.